grammar_to_md.py

```diff
@@ -78,7 +78,8 @@
               'BCONST'         : '<i>binary-literal</i>',
               'FCONST'         : '<i>float-literal</i>',
               'ELLIPSIS'       : '<i>"..."</i>',
-              '<empty>'        : '&lt;empty&gt;'
+              '<empty>'        : '&lt;empty&gt;',
+              '_'              : '<b>_</b>'
```

Comment:
I prefer the token be named `DISCARDREF`.

Reply:
Disagree weakly. Other keyword-formed tokens are named as the keyword.